src/components/blocks/wrappers/LocalLanguageAlternatives.tsx

```diff
@@ -35,7 +35,20 @@ const LocalLanguageAlternatives = ({
   }, [initialData]);
 
   const setLocalSelected = (value: string) => {
-    setSelected(data ? data[value] : '');
+    const selectedLocalData = data ? data[value] : '';
+    if (selectedLocalData != null && isObject(selectedLocalData[0])) {
```

Comment:
Can you do `isObject(selectedLocalData?.[0])`? Wary of using `!=` over `!==` generally.

Reply:
@MymmiJ  thanks just updated but I didn't removed the `selectedLocalData != null` cos the `selectedLocalData[0]?.attribs?.lang` is not happy